srcpkgs/direwolf/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'direwolf'
+pkgname=direwolf
+version=1.6
+revision=1
+build_style=cmake
+archs="~*-musl"
```

Comment:
Not really an option, this is simple enough to fix.

If you want to do it easily, just patch https://github.com/wb2osz/direwolf/blob/8ac14f86f5ee43176423fbdb07fe65aa18fcdb81/src/direwolf.h#L285 to `#if ! defined(__GLIBC__) || defined(__OpenBSD__) || defined(__FreeBSD__) || defined(__APPLE__)`

Reply:
Oops sorry i already pushed before i read your comments. Anyway thanks for the fast replys i will try that.

Reply:
My 2 cents:
I would use:
```c
#if (defined(__linux__) && !defined(__GLIBC__)) || defined(__OpenBSD__) || defined(__FreeBSD__) || defined(__APPLE__)defined(__APPLE__)
```

With this form, we can submit to upstream.

Reply:
@sgn that one is still encoding stuff that it doesn't need. They use CMake, they can see if the function exists at configure time. I just made the minimal patch that'd work for us :P 

See https://github.com/wb2osz/direwolf/commit/1d67b446694e122552b4e7387850e4e9a23d48d3 for the version in the dev branch. If @DavidKaep could test this out, it'd be great.